src/cpu/minor/fetch2.cc

```diff
@@ -424,6 +425,7 @@ Fetch2::evaluate()
                         stats.fpInstructions++;
                     else if (decoded_inst->isInteger())
```

Comment:
tangentially related to your commit, but we should probably add brackets {} around the if and else if so someone doesn't think your change on line 428 is supposed to be part of the else if.